man/systemd.network.xml

```diff
@@ -3183,6 +3183,31 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
     </variablelist>
     </refsect1>
 
+    <refsect1>
+    <title>[IPv6PREF64Prefix] Section Options</title>
+    <para>One or more [IPv6PREF64Prefix] sections contain the IPv6 PREF (or NAT64) prefixes that are announced via Router
+    Advertisements. See <ulink url="https://tools.ietf.org/html/rfc8781">RFC 8781</ulink> for further
+    details.</para>
+
+    <variablelist class='network-directives'>
+
+      <varlistentry>
+        <term><varname>Prefix=</varname></term>
+
+        <listitem><para>The IPv6 PREF64 (or NAT64) prefix that is to be distributed to hosts. The setting holds
+        an IPv6 prefix that should be set up for NAT64 translation (PLAT) to allow 464XLAT on the network segment.
+        <literal>/</literal> character. Use multiple [IPv6PREF64Prefix] sections to configure multiple IPv6
+        prefixes since prefix lifetimes, prefixes since prefix lifetimes may differ from one prefix to another..</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>ValidLifetimeSec=</varname></term>
+        <listitem><para>Valid lifetimes for the prefix measured in seconds.
```

Comment:
```suggestion
        <listitem><para>Valid lifetime for the prefix measured in seconds. Should be greater than or equal `RouterLifetimeSec=`.
```

Not sure about the lingo, but this point is somewhat important.

> It would be highly undesirable for the NAT64 prefix to have a lifetime shorter than the Router Lifetime, which is defined in [Section 4.2](https://www.rfc-editor.org/rfc/rfc4861#section-4.2) of [[RFC4861](https://datatracker.ietf.org/doc/html/rfc4861)] as a 16-bit unsigned integer. If the NAT64 prefix lifetime is not at least equal to the default Router Lifetime, it might lead to scenarios in which the NAT64 prefix lifetime expires before the arrival of the next unsolicited RA.

https://datatracker.ietf.org/doc/html/rfc8781#name-scaled-lifetime-processing